bundles/mapping/mapmodule/plugin/logo/LogoPlugin.js

```diff
@@ -367,6 +368,33 @@ Oskari.clazz.define(
                     'source' : indicators[id].organization
                 });
             });
+        },
+        /**
+         * @method addContentFromService
```

Comment:
Comment mismatch to actual method name. Is updateLabels() a better name for this?